cmd/harborAPI.go

```diff
@@ -561,7 +561,7 @@ func Restart(username string, token string, shipment string, env string) {
 	values := make(map[string]interface{})
 	values["shipment"] = shipment
 	values["env"] = env
-	template, _ := uritemplates.Parse(config.ShipitURI + "/v1/shipment/{shipment}/environment/{env}/envVar/RESTART")
+	template, _ := uritemplates.Parse(config.ShipitURI + "/v1/shipment/{shipment}/environment/{env}/envVar/HC_RESTART")
```

Comment:
yeah. well, might be better to know where they restarted it from? I know it's not super important, but could possibly add more HC things in the future. Would be good to Namespace just this client.